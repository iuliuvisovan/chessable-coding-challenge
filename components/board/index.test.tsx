import React from 'react';
import { create } from 'react-test-renderer';
import Board from './index';

const boardTheme = { darkSquareColor: '#b98661', lightSquareColor: '#f4d9b7' };

const tree = create(<Board theme={boardTheme} />);

test('snapshot', () => {
  expect(tree).toMatchSnapshot();
});
