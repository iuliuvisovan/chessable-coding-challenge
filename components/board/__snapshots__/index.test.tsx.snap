// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<View
  style={
    Array [
      Object {
        "borderWidth": 1,
        "flexDirection": "row",
        "flexWrap": "wrap",
        "height": 322,
        "width": 322,
      },
      Object {
        "borderColor": "#b98661",
      },
    ]
  }
>
  <View
    style={
      Array [
        Object {
          "height": 40,
          "width": 40,
        },
        Object {
          "backgroundColor": "#f4d9b7",
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "height": 40,
          "width": 40,
        },
        Object {
          "backgroundColor": "#b98661",
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "height": 40,
          "width": 40,
        },
        Object {
          "backgroundColor": "#f4d9b7",
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "height": 40,
          "width": 40,
        },
        Object {
          "backgroundColor": "#b98661",
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "height": 40,
          "width": 40,
        },
        Object {
          "backgroundColor": "#f4d9b7",
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "height": 40,
          "width": 40,
        },
        Object {
          "backgroundColor": "#b98661",
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "height": 40,
          "width": 40,
        },
        Object {
          "backgroundColor": "#f4d9b7",
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "height": 40,
          "width": 40,
        },
        Object {
          "backgroundColor": "#b98661",
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "height": 40,
          "width": 40,
        },
        Object {
          "backgroundColor": "#b98661",
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "height": 40,
          "width": 40,
        },
        Object {
          "backgroundColor": "#f4d9b7",
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "height": 40,
          "width": 40,
        },
        Object {
          "backgroundColor": "#b98661",
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "height": 40,
          "width": 40,
        },
        Object {
          "backgroundColor": "#f4d9b7",
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "height": 40,
          "width": 40,
        },
        Object {
          "backgroundColor": "#b98661",
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "height": 40,
          "width": 40,
        },
        Object {
          "backgroundColor": "#f4d9b7",
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "height": 40,
          "width": 40,
        },
        Object {
          "backgroundColor": "#b98661",
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "height": 40,
          "width": 40,
        },
        Object {
          "backgroundColor": "#f4d9b7",
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "height": 40,
          "width": 40,
        },
        Object {
          "backgroundColor": "#f4d9b7",
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "height": 40,
          "width": 40,
        },
        Object {
          "backgroundColor": "#b98661",
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "height": 40,
          "width": 40,
        },
        Object {
          "backgroundColor": "#f4d9b7",
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "height": 40,
          "width": 40,
        },
        Object {
          "backgroundColor": "#b98661",
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "height": 40,
          "width": 40,
        },
        Object {
          "backgroundColor": "#f4d9b7",
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "height": 40,
          "width": 40,
        },
        Object {
          "backgroundColor": "#b98661",
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "height": 40,
          "width": 40,
        },
        Object {
          "backgroundColor": "#f4d9b7",
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "height": 40,
          "width": 40,
        },
        Object {
          "backgroundColor": "#b98661",
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "height": 40,
          "width": 40,
        },
        Object {
          "backgroundColor": "#b98661",
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "height": 40,
          "width": 40,
        },
        Object {
          "backgroundColor": "#f4d9b7",
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "height": 40,
          "width": 40,
        },
        Object {
          "backgroundColor": "#b98661",
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "height": 40,
          "width": 40,
        },
        Object {
          "backgroundColor": "#f4d9b7",
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "height": 40,
          "width": 40,
        },
        Object {
          "backgroundColor": "#b98661",
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "height": 40,
          "width": 40,
        },
        Object {
          "backgroundColor": "#f4d9b7",
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "height": 40,
          "width": 40,
        },
        Object {
          "backgroundColor": "#b98661",
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "height": 40,
          "width": 40,
        },
        Object {
          "backgroundColor": "#f4d9b7",
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "height": 40,
          "width": 40,
        },
        Object {
          "backgroundColor": "#f4d9b7",
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "height": 40,
          "width": 40,
        },
        Object {
          "backgroundColor": "#b98661",
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "height": 40,
          "width": 40,
        },
        Object {
          "backgroundColor": "#f4d9b7",
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "height": 40,
          "width": 40,
        },
        Object {
          "backgroundColor": "#b98661",
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "height": 40,
          "width": 40,
        },
        Object {
          "backgroundColor": "#f4d9b7",
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "height": 40,
          "width": 40,
        },
        Object {
          "backgroundColor": "#b98661",
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "height": 40,
          "width": 40,
        },
        Object {
          "backgroundColor": "#f4d9b7",
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "height": 40,
          "width": 40,
        },
        Object {
          "backgroundColor": "#b98661",
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "height": 40,
          "width": 40,
        },
        Object {
          "backgroundColor": "#b98661",
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "height": 40,
          "width": 40,
        },
        Object {
          "backgroundColor": "#f4d9b7",
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "height": 40,
          "width": 40,
        },
        Object {
          "backgroundColor": "#b98661",
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "height": 40,
          "width": 40,
        },
        Object {
          "backgroundColor": "#f4d9b7",
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "height": 40,
          "width": 40,
        },
        Object {
          "backgroundColor": "#b98661",
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "height": 40,
          "width": 40,
        },
        Object {
          "backgroundColor": "#f4d9b7",
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "height": 40,
          "width": 40,
        },
        Object {
          "backgroundColor": "#b98661",
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "height": 40,
          "width": 40,
        },
        Object {
          "backgroundColor": "#f4d9b7",
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "height": 40,
          "width": 40,
        },
        Object {
          "backgroundColor": "#f4d9b7",
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "height": 40,
          "width": 40,
        },
        Object {
          "backgroundColor": "#b98661",
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "height": 40,
          "width": 40,
        },
        Object {
          "backgroundColor": "#f4d9b7",
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "height": 40,
          "width": 40,
        },
        Object {
          "backgroundColor": "#b98661",
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "height": 40,
          "width": 40,
        },
        Object {
          "backgroundColor": "#f4d9b7",
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "height": 40,
          "width": 40,
        },
        Object {
          "backgroundColor": "#b98661",
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "height": 40,
          "width": 40,
        },
        Object {
          "backgroundColor": "#f4d9b7",
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "height": 40,
          "width": 40,
        },
        Object {
          "backgroundColor": "#b98661",
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "height": 40,
          "width": 40,
        },
        Object {
          "backgroundColor": "#b98661",
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "height": 40,
          "width": 40,
        },
        Object {
          "backgroundColor": "#f4d9b7",
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "height": 40,
          "width": 40,
        },
        Object {
          "backgroundColor": "#b98661",
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "height": 40,
          "width": 40,
        },
        Object {
          "backgroundColor": "#f4d9b7",
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "height": 40,
          "width": 40,
        },
        Object {
          "backgroundColor": "#b98661",
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "height": 40,
          "width": 40,
        },
        Object {
          "backgroundColor": "#f4d9b7",
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "height": 40,
          "width": 40,
        },
        Object {
          "backgroundColor": "#b98661",
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "height": 40,
          "width": 40,
        },
        Object {
          "backgroundColor": "#f4d9b7",
        },
      ]
    }
  />
</View>
`;
